ydb/library/actors/core/event_pb.h

```diff
@@ -188,23 +194,15 @@ namespace NActors {
         }
 
         bool SerializeToArcadiaStream(TChunkSerializer* chunker) const override {
-            return SerializeToArcadiaStreamImpl(chunker, TString());
+            if (!SerializeToArcadiaStreamImpl(chunker, Payload)) 
```

Comment:
we use braces even with single statement construct

also, you can write it in single expression -- return SerializeToArcadiaStreamImpl(...) && Record.SerializeToZeroCopyStream(...);